include/GL/glui.h

```diff
@@ -581,7 +642,7 @@ class GLUIAPI GLUI_Glut_Window : public GLUI_Node
     void (*glut_mouse_CB)(int,int,int,int);
     void (*glut_visibility_CB)(int);
     void (*glut_motion_CB)(int,int);
-    void (*glut_display_CB)(void);
```

Comment:
I don't mean to be rude. It just cannot be part of my calculus. I think it's better for everyone to value each other's time as much as possible. To be blunt I think a lot of open-source development is dysfunctional and unfruitful and given the choice I just go off and do my own thing. If cannot be productive. And not marginally productive, but more productive than the alternative (which is doing my own thing by myself.)

The sooner we reach an understanding the better. I regret the (void) changes because it ultimately got in the way of the set_glutXFunc diffs, and in general. But I did not think it would be an article of discussion. I can't do piecemeal changes, but that's definitely something that could've waited for the reformat.